pkgs/development/tools/go-task/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl }:
+
+let
+  targetPath = "$out/bin";
+
+in
+stdenv.mkDerivation {
+  pname   = "go-task";
+  version = "2.8.1";
+
+  src = fetchurl {
+    url    = "https://github.com/go-task/task/releases/download/v2.8.1/task_linux_amd64.tar.gz";
+    sha256 = "1fhshi5y7lg479b2lg02swc7alwwgvn1gm4h9yq2bnx6hppnkjn7";
+  };
+
+  unpackPhase = ''
+    tar -xzf $src
+  '';
+
+  installPhase = ''
+    mkdir -p ${targetPath}
+    mv ./task ${targetPath}
+  '';
+
+  meta = {
+    homepage    = "https://taskfile.dev/";
+    description = "A task runner / simpler Make alternative written in Go";
+    license     = stdenv.lib.licenses.mit;
```

With the change above, you can change this to:

```suggestion
    license     = licenses.mit;
```

---

done :+1: